sunpy/net/dataretriever/sources/rhessi.py

```diff
@@ -2,18 +2,17 @@
 # This module was developed under funding provided by
 # Google Summer of Code 2014
 import socket
-import posixpath
 from datetime import datetime
 from urllib.error import URLError
 from urllib.request import urlopen, urlretrieve
 
-from dateutil.rrule import rrule, MONTHLY
+import posixpath
```

Comment:
errr, is this not stdlib?!

Reply:
isort thinks not

Reply:
Really odd here, if i run isort via the precommit, this is in the correct place, if i run it locally it is not.